src/pymor/reductors/h2.py

```diff
@@ -103,9 +103,14 @@ def _order_to_sigma_b_c(self, r):
     @staticmethod
     def _rom_to_sigma_b_c(rom, force_sigma_in_rhp):
         poles, b, c = _lti_to_poles_b_c(rom)
-        sigma = (np.abs(poles.real) + poles.imag * 1j
-                 if force_sigma_in_rhp
-                 else -poles)
+        if rom.sampling_time > 0:
+            sigma = 1/np.conjugate(poles)
+            if force_sigma_in_rhp:
```

Comment:
In general it is good to do assertions that rely only on input parameters at the beginning of the method in order to avoid unecessary computations.

Reply:
Sorry, I was not very clear. I meant in `TFIRKAReductor` :D